apis/Google.Cloud.Diagnostics.Common/Google.Cloud.Diagnostics.Common/project.json

```diff
@@ -18,7 +18,7 @@
   },
 
   "dependencies": {
-    "Google.Api.Gax": "1.0.0-beta05",
+    "Google.Api.Gax": "1.0.0-beta06",
```

Comment:
Does the version number of this library need to change from alpha01 to alpha02/beta01?